### DISCUSSION TITLE
Ulcer

### DISCUSSION THREAD

**Discussion by InChil2** [InChil2/Detection_of_lesions#20] [전처리 방안]
시작

**Comment by pkwon35** [comment]
# <Ulcer>
##"Bleeding and Ulcer Detection Using   Wireless Capsule Endoscopy Images"

https://www.researchgate.net/publication/276044203_Bleeding_and_Ulcer_Detection_Using_Wireless_Capsule_Endoscopy_Images



### 아래와 같이 Greyscale / binary 적용 시 이렇게 바뀐다.

![image](https://user-images.githubusercontent.com/25214976/142855381-b67a2872-64a1-471a-9dc0-da0813904b10.png)


<Ulcer>
"The ulcer area is usually white or pale yellow. In this study, the binarized ulcer area was rendered white,"

- The first step in ulcer detection is transforming the original image (Figure 4(a)) into a **grayscale image** (Figure 4(b)) that is **binarized** based on a predefined threshold


 The classification performance was considerably improved after applying the **color saturation threshold**. The most favorable combination of parameters for automatically detecting bleeding and ulcers is listed in Table 9. 

정리:
1) Ulcer 는 보통 흰색/연한노랑색. 이진화 적용시 ulcer 부분은 흰색으로 도드라짐. 
   - 변환 : original -> grayscale -> binarized

**Comment by pkwon35** [comment]
[ulcer(Countourlet_transform,Log_Gabor_filter,saliency_map).pdf](https://github.com/InChil2/Detection_of_lesions/files/7581268/ulcer.Countourlet_transform.Log_Gabor_filter.saliency_map.pdf)



"ulcer detection in WCE images. The proposed method combines the merits of
both **Contourlet transform** and **Log Gabor filter**."

**Reply by pkwon35** [reply]
<LogGabor 필터 참고 코드>

https://www.kaggle.com/ashuta03/image-texture-segmentation-using-gabor-filters#Pre-processing

전
![image](https://user-images.githubusercontent.com/25214976/142874168-c2ef94c5-7e11-4a91-9ef4-4ab6fdfd9304.png)



후
![image](https://user-images.githubusercontent.com/25214976/142874201-da07ac6d-4e3c-4bc4-a7a9-bd16f3106ab3.png)
